uber/templates/regform.html

```diff
@@ -78,7 +78,11 @@
     <script type="text/javascript">
         var donationChanged = function () {
             setVisible('.affiliate-row', $.val('amount_extra') > 0);
```

Comment:
Always show the tshirt row for people with staff badges.  Again, seems like a sensible enough default to belong in the core repo.